azure/notebooks/azure_mnmg.ipynb

```diff
@@ -69,16 +71,15 @@
    "outputs": [],
    "source": [
     "location = \"SOUTH CENTRAL US\"\n",
-    "resource_group = \"RAPIDS-HPO-test\"\n",
+    "resource_group = \"RAPIDS-mnmg\"\n",
     "vnet = \"dask-vnet\"\n",
-    "security_group = \"test-security-group\"\n",
+    "security_group = \"dask-nsg\"\n",
     "\n",
     "vm_size = \"Standard_NC12s_v3\"\n",
-    "docker_image = \"rapidsai/rapidsai-core:cuda10.2-runtime-ubuntu18.04-py3.8\"\n",
+    "docker_image = \"rapidsai/rapidsai-nightly:0.18-cuda10.2-runtime-ubuntu18.04-py3.8\"\n",
```

Comment:
```suggestion
    "docker_image = \"rapidsai/rapidsai-nightly:0.18-cuda11.0-runtime-ubuntu18.04-py3.8\"\n",
```

Reply:
Do we need to modify the README to use the same container?

Reply:
Yes, thank you. I missed that.